infra/aws/terraform/prow-build-cluster/resources/kubecost/README.md

```diff
@@ -1,120 +0,0 @@
-# Kubecost Installation
-
-Some manual actions need to be taken before Flux takes over the process.
-
-## AWS Preperations
-
-Kubecost requires some configuration on AWS side:
-
-<details>
-  <summary>policy-kubecost-aws-s3.json</summary>
-
-  ```json
-  {
-      "Version": "2012-10-17",
-      "Statement": [
-          {
-              "Sid": "VisualEditor0",
-              "Effect": "Allow",
-              "Action": [
-                  "s3:ListBucket",
-                  "s3:GetBucketLocation"
-              ],
-              "Resource": "arn:aws:s3:::kubecost-prow-clusters-metrics"
-          },
-          {
-              "Sid": "VisualEditor1",
-              "Effect": "Allow",
-              "Action": [
-                  "s3:PutObject",
-                  "s3:GetObject",
-                  "s3:ListBucketMultipartUploads",
-                  "s3:AbortMultipartUpload",
-                  "s3:ListBucket",
-                  "s3:DeleteObject",
-                  "s3:ListMultipartUploadParts"
-              ],
-              "Resource": [
-                  "arn:aws:s3:::kubecost-prow-clusters-metrics",
-                  "arn:aws:s3:::kubecost-prow-clusters-metrics/*"
-              ]
-          }
-      ]
-  }
-  ```
-</details>
-
-Create the IAM policy and the namespace:
-
-```bash
-aws iam create-policy \
- --policy-name kubecost-s3-federated-policy \
- --policy-document file://policy-kubecost-aws-s3.json
-
-kubectl create namespace kubecost
-```
-
-Create a secret to access to the S3 bucket for the federation. 
-
-> [!IMPORTANT]
-> The `federated-store.yaml` must be updated with the correct AWS credentials
-
-<details>
-  <summary>federated-store.yaml</summary>
-
-  ```yaml
-  type: S3
-  config:
-    bucket: "kubecost-prow-clusters-metrics"
```

Comment:
@koksay Should we also delete this bucket?

Reply:
Yes, I had done a couple of manual steps, I will revert them all.